locales/en/fix.ftl

```diff
@@ -0,0 +1,200 @@
+ssn-modal-description-freeze-credit-part-one = <b>Freezing your credit</b> prevents anyone from opening a new account in your name. It’s free and won’t negatively affect your credit score, but you’ll need to unfreeze it before opening any new accounts.
+ssn-modal-description-freeze-credit-part-two = To freeze your credit, contact each of the three credit bureaus — <equifax_link>Equifax</equifax_link>, <experian_link>Experian</experian_link>, and <transunion_link>TransUnion</transunion_link>.
+ssn-modal-learn-more = Learn more about fraud alerts and credit freezes
+ssn-modal-ok = OK
+
+# PIN Breaches
+
+high-risk-breach-pin-title = Your PIN was exposed
+high-risk-breach-pin-description = Taking action ASAP could give you more legal protections to help you recover any losses.
+high-risk-breach-pin-step-one = Notify your bank immediately that your PIN has been compromised.
+high-risk-breach-pin-step-two = Change your PIN anywhere you’ve used the same one.
+high-risk-breach-pin-step-three = Check your accounts for unauthorized charges.
+
+# No high risk breaches found
+
+high-risk-breach-none-title = Great news, we didn’t find any high risk data breaches
+# Variables
+# $email_list is list of emails that the user is monitoring for breaches. E.g. john@yahoo.com, ali@gmail.com, sam@hotmail.com
+high-risk-breach-none-description = We detect data breaches based on your email address, and we didn’t find any high risk data breaches for { $email_list }.
+high-risk-breach-none-sub-description-part-one = High risk data breaches include:
+high-risk-breach-none-sub-description-ssn = Social security number
+high-risk-breach-none-sub-description-bank-account = Bank account info
+high-risk-breach-none-sub-description-cc-number = Credit card numbers
+high-risk-breach-none-sub-description-pin = PINs
+high-risk-breach-none-continue = Continue
+
+# Security recommendations
+
+security-recommendation-steps-label = Security recommendations
+security-recommendation-steps-title = Here’s our advice:
+security-recommendation-steps-cta-label = Got it!
+
+# Phone security recommendation
+
+security-recommendation-phone-title = Protect your phone number
+# $num_breaches is the number of breaches where the phone number was found.
+security-recommendation-phone-summary = { $num_breaches ->
+  [one] Your phone number was exposed in { $num_breaches } data breach:
+  *[other] Your phone number was exposed in { $num_breaches } data breaches:
+}
+security-recommendation-phone-description = Unfortunately you can’t take it back. But there are steps you can take to make sure you stay safe.
+security-recommendation-phone-step-one = Block spam numbers to prevent more junk calls
+security-recommendation-phone-step-two = Don’t click on links in texts from unknown senders; if it appears to be from a trusted source, call directly to confirm
+security-recommendation-phone-step-three = Use a <link_to_info>{ -brand-relay } phone mask</link_to_info> to protect your phone in the future
```

Comment:
Phone masks are not available out of US, is this hidden?

Reply:
[It's not clear if it should be](https://docs.google.com/document/d/1-_KSIEW07pbT3eO-bS4JcnrTstCduucc1omdo2RXzuM/edit?disco=AAABDlbwnZw), but I've hidden it for now: 3b7cd9dee.

It might be reverted though, since there is the ability to sign up for the waiting list.